apps/src/templates/SchoolZipSearch.jsx

```diff
@@ -40,8 +55,16 @@ export default function SchoolZipSearch({fieldNames, zip, disabled}) {
             error.message
           );
         });
+      setIsSchoolDropdownDisabled(false);
```

Comment:
I wonder if it makes sense to set this to false when schools are loaded 🤔

Reply:
Otherwise, I'm wondering if this should be in state at all. Checking a regex could be done on every render

Reply:
We had a conversation about this offline- to keep from having a duplicate regex check, we opted to leave the structure as-is.